base/notify-api/api-deployment.yaml

```diff
@@ -300,12 +300,12 @@ spec:
             periodSeconds: 3
             timeoutSeconds: 1
             successThreshold: 3
-            failureThreshold: 5
+            failureThreshold: 10
```

Comment:
Once the startup probe is done, the readiness and liveness probes will kick in. The readiness will determine when the app is ready and might need to wait more time than what the startup probe allowed. The reason for that is that the first http hit will trigger an initialization for lazy loaded components such as New Relic. So we need to give it more flexibility. 

A failure threshold of 10 for 3 period seconds means that a total of 30 seconds will be tried, but after waiting for an initial delay seconds of 5 seconds. Hence 35 seconds is allowed for this readiness stage.